src/main/resources/standardTestCases/refresh_token.test

```diff
@@ -1,5 +1,6 @@
 name: refresh_token
 testFormat: workflowsuccess
+backend: JES
```

Comment:
You're right, however that conf file doesn't seem to be used at the moment. let's chat in person, i'm probably confused